aries_cloudagent/wallet/models/wallet_record.py

```diff
@@ -66,7 +67,7 @@ def wallet_type(self) -> str:
         return self.settings.get("wallet.type")
 
     @property
-    def wallet_webhook_urls(self) -> str:
+    def wallet_webhook_urls(self) -> list:
```

Comment:
Should likely be `Sequence[str]` not `list`

Reply:
Thanks for your reviews. It is fixed.